templates/zerver/integrations.html

```diff
@@ -1179,6 +1185,31 @@
 
     </div>
 
+    <div id="pingdom" class="integration-instructions">
+
+      <p>Zulip supports Pingdom integration and can notify you of
+          uptime status changes from your Pingdom dashboard.</p>
+
+      <p>First, create the stream you'd like to use for Pingdom notifications,
+          and subscribe all interested parties to this stream. We
+          recommend the name <code>pingdom</code>.</p>
+
+      <p><code>{{ external_api_uri }}/v1/external/pingdom?api_key=abcdefgh&amp;stream=pingdom</code></p>
+
+      <p>where <code>api_key</code> is the API key of your Zulip bot,
+      and <code>stream</code> is the stream name you want the
+      notifications sent to.</p>
+
+      <p>Next, under following url: </p>
+      <p><code>https://my.pingdom.com/reports/integration/settings</code></p>
+      <p>create your integration by clicking on <code>Add Integration</code> button and filling form as following:</p>
+      <img class="screenshot" src="/static/images/integrations/pingdom/001.png" />
+
+      <p>Last, during creating or editing your check, scroll down to <code>Connect Integrations</code>
+         section and check your integration </p>
```

Comment:
I'm not sure I understand this step; what are they supposed to do to check the integration?